docs/sections/examples.md

```diff
@@ -6,7 +6,30 @@
 
 ### Using Virtualenv (Python)
 
-**TODO**
+Python provides the ability of recreating and isolating environments
+with all the dependencies that are needed by an application that is
+written in this language.
+
+This tool is [virtualenv](https://virtualenv.pypa.io/en/stable/) and it can be used 
+to create an isolated environment with all the dependencies of a python application,
+including the version of the python runtime itself. 
+This is a lightweight way of creating portable pipelines.
+
+Popper pipelines automate and create an explicit record of the steps
+that need to be followed in order to create these isolated environments.
+
+Examples:
+
+[sea-surface-mapping](https://github.com/popperized/swc-lesson-pipelines/tree/master/pipelines/sea-surface-mapping)
+In this case, Popper helps make sure that a virtualenv is enabled and 
+running at the time of execution by throwing an exception if your 
+virtualenv is disabled.
+
+Or if you want Popper to make the virtualenv for you, you can check the following pipeline:
```

Comment:
This might give the impression that popper is doing the work but actually the pipeline is the one that codifies the logic of creating the virtual environment. So maybe reword to something like: "Another alternative is to create a virtual environment as part of the execution of a pipeline. The `validator` pipeline ..."